db/changes.go

```diff
@@ -672,6 +672,11 @@ func (col *DatabaseCollectionWithUser) SimpleMultiChangesFeed(ctx context.Contex
 			channelsSince = channels.AtSequence(chans, 0)
 		}
 
+		if options.RequestPlusSeq > 0 {
+			// QUESTION: why can't I use 0 for AtSequence here?
+			channelsSince.Add(channels.AtSequence(base.SetOf(unusedSeqKey), currentCachedSequence))
```

Comment:
I don't think you actually want to add the pseudo-channel to channelsSince, as that variable is used for more than just the call to changeWaiter.UpdateChannels().  You're right that changeWaiter doesn't use the AtSequence so it could be ignored, but by adding it to channelsSince you'd be doing things we don't want/need to do, like adding a channel feed on line 721/726 below.   

To avoid allocating a completely new set just to pass it to changeWaiter.UpdateChannels, I'm fine with modifying the changeWaiter.UpdateChannels add an optional parameter that would be a list of 'pseudo-channels'.  We wouldn't need to store those differently inside the changeWaiter (yet).